### ISSUE TITLE
Unable to whilelist with ipv6

### ISSUE THREAD

**Issue by ramaraosrikakulapu** [Enterprise-connect/ec-x-sdk#67]
Need to readdress ip filtering with IPv6 addresses.

**Comment by ayasuda2OO3** [comment]
Please provide logs. @ramaraosrikakulapu

**Comment by ramaraosrikakulapu** [comment]
Here are the gateway logs with IPv6 address -

```
2020-02-20T13:08:19.28-0500 [APP/PROC/WEB/0] OUT  [EC Debug] 2020/02/20 18:08:19 []interface {}=>[165.156.40.27, 10.72.131.166]
   2020-02-20T13:08:19.28-0500 [APP/PROC/WEB/0] OUT  [EC Gateway] 2020/02/20 18:08:19 [the request port might not be specified. continuing validation..]
   2020-02-20T13:08:19.28-0500 [APP/PROC/WEB/0] OUT  [EC Gateway] 2020/02/20 18:08:19 [the request port might not be specified. continuing validation..]
   2020-02-20T13:08:19.28-0500 [APP/PROC/WEB/0] OUT  [EC Gateway] 2020/02/20 18:08:19 [ip passed filtering with x-forward-for.]
   2020-02-20T13:08:19.28-0500 [APP/PROC/WEB/0] OUT  [EC Debug] 2020/02/20 18:08:19 []interface {}=>[[127.0.0.1 ::1 fe80::1 3.60.34.51 fe80::10e0:8fff:fed2:5b70 fe80::2ac6:ed08:8ae0:5253 172.16.150.1 172.16.154.1]]
   2020-02-20T13:08:19.28-0500 [APP/PROC/WEB/0] OUT  [EC Gateway] 2020/02/20 18:08:19 [ip passed filtering with ec-options header.]
   2020-02-20T13:08:19.28-0500 [APP/PROC/WEB/0] OUT  [EC Debug] 2020/02/20 18:08:19 []interface {}=>[_rconf.GatewayHost.Host digital-poc-gateway.run.aws-usw02-dev.ice.predix.io GatewayHost:Host {https   digital-poc-gateway.run.aws-usw02-dev.ice.predix.io   false  }]
   2020-02-20T13:08:19.28-0500 [APP/PROC/WEB/0] OUT  [EC Debug] 2020/02/20 18:08:19 []interface {}=>[https://3521b371-4c06-4d29-abee-83c3af64264a.run.aws-usw02-dev.ice.predix.io/v1beta/api/token/validate]
```

**Comment by ayasuda2003** [comment]
Is the server use or behind a proxy setting? @ramaraosrikakulapu

**Comment by ayasuda2OO3** [comment]
In team discussion if including the local ip addresses in the white/black list still make sense and a practice to follow.

**Comment by ayasuda2OO3** [comment]
Closed. Issue has been addressed in [release v1.1beta.2730](https://github.com/Enterprise-connect/sdk/releases/tag/v1.1beta.fukuoka.2730)
